includes/class-wp-feature-schema-adapter.php

```diff
@@ -354,53 +354,6 @@ private function rule_strict_object_encoding( $rule_name, $data ) {
 		return $data;
 	}
 
-	/**
```

Comment:
The proper fix is at https://github.com/Automattic/wp-feature-api/pull/21 and a follow-up change. This is only temporarily added for testing.